test/packages/custom_logs/manifest.yml

```diff
@@ -0,0 +1,46 @@
+format_version: 1.0.0
+name: custom_logs
+title: Custom Logs
+description: >-
+  Collect custom logs with Elastic Agent.
+type: input
+version: 1.2.3
+release: ga
+license: basic
+categories:
+  - custom
+policy_templates:
+  - name: first_policy_template
```

Comment:
I didn't put the `elasticsearch` block here. Should I add few/all options from [here](https://github.com/elastic/package-spec/blob/ff81936f9495f64c26722314e5e86355f575f92f/versions/1/integration/data_stream/manifest.spec.yml#L179)?

Reply:
Yeah, I wouldn't see a reason not to support the same options. Unless there is something we want to deprecate from the data stream manifests.

Reply:
So, we have:

`elasticsearch.index_template.settings`
`elasticsearch.index_template.mappings`
`elasticsearch.index_template.ingest_pipeline.name`
`elasticsearch.privileges.indices`

As we don't know in advance what would be the target index/ingest pipeline, we can't set these properties:
`elasticsearch.index_template.ingest_pipeline.name`
`elasticsearch.privileges.indices`

Am I right?

Reply:
> As we don't know in advance what would be the target index/ingest pipeline, we can't set these properties:
> `elasticsearch.index_template.ingest_pipeline.name`
> `elasticsearch.privileges.indices`

Umm, I don't see these are used in current integrations repo. What are use cases for these options in integration packages? Could they apply in input packages?

In any case, maybe you are right with not adding the block by now, so we start small and we see later possible use cases.

Reply:
> Umm, I don't see these are used in current integrations repo. What are use cases for these options in integration packages? Could they apply in input packages?

If you can find any package here, then most likely it's APM or Endpoint.